.gitignore

```diff
@@ -15,3 +15,5 @@ __pycache__
 .cache
 .local
 .pre-commit-config.yaml
+.vscode/
+.python-version
```

Comment:
What is `.python-version`? I've never encountered this before.

Reply:
It's created by `pyenv local`

I'd recommend @johnzeringue and other pyenv uses add it to their global gitignore

Reply:
Okay I removed this line